src/extensions/default/AutoUpdate/main.js

```diff
@@ -421,15 +378,35 @@ define(function (require, exports, module) {
      * @param {Array} updates - array object containing info about updates
```

Comment:
@mbhavi Please create a JSDoc custom node definition for update entry.

Reply:
@swmitra , addressed in 44fef7e